### ISSUE TITLE
[frontend] new alt text cannot be scrolled

### ISSUE THREAD

**Issue by fluffye-yay** [gabboman/wafrn#220]
alt text that is too long to fit within bounds of image is cut off instead of being scrollable, note bottoms of letters visible at top of alt text

![Screenshot_20241231-171735](https://github.com/user-attachments/assets/6a92435e-95ba-42d7-984e-9ae50fbf71f8)

https://app.wafrn.net/fediverse/post/3d18f052-86b9-42bc-95dd-8b79452f09ad

**Comment by FireIsGood** [comment]
Aw dangit

**Comment by FireIsGood** [comment]
Ok new pulls soon&trade; fixes this along with adding some other stuff

![image](https://github.com/user-attachments/assets/fd695eaf-9c72-4050-846f-f464fdee501a)

**Comment by fluffye-yay** [comment]
works properly now tdanks
